### PR TITLE
Final fix for Commit Checker GH Action

### PR DESCRIPTION
**Pull request by rimolive**
**Description of your changes:**
Fixes the Commit Checker GitHub Action to checkout the PR branch before running the check.

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by hbelmiro — COMMENTED** [review]
/lgtm

**Comment by dsp-developers** [comment]
### Commit Checker results:

```
**NOTE**: These are the results of the commit checker scans. 
If these are not commits from upstream kfp, then please ensure
you adhere to the commit checker formatting
```
commitchecker verson unknown
ERROR: couldn't find commits from effca01ddbe891dc2f1badfb01f30bf20283f3ef..fbb2c121655f10f366449febcdf8f669c4204a88: error executing git log: fatal: Invalid revision range effca01ddbe891dc2f1badfb01f30bf20283f3ef..fbb2c121655f10f366449febcdf8f669c4204a88
: exit status 128

File: .github/workflows/commit-check-pr.yml [hunk 1]
```diff
@@ -6,8 +6,6 @@ on:
       - completed
```
Comment (DharmitD) [inline .github/workflows/commit-check-pr.yml:6]: minor nitpick but `version` and `commit checker` are spelled incorrectly in the commit checker error message: https://github.com/rimolive/data-science-pipelines/pull/2#issuecomment-2142734925

could you add that fix as part of this PR?

**Comment by dsp-developers** [comment]
A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-50`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-50`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-50`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-50`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-50`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-50`

**Comment by dsp-developers** [comment]
An OCP cluster where you are logged in as cluster admin is required.

The Data Science Pipelines team recommends testing this using the Data Science Pipelines Operator.  Check [here](https://github.com/opendatahub-io/data-science-pipelines-operator) for more information on using the DSPO.

To use and deploy a DSP stack with these images (assuming the DSPO is deployed), first save the following YAML to a file named `dspa.pr-50.yaml`:
```yaml
apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
kind: DataSciencePipelinesApplication
metadata:
  name: pr-50
spec:
  dspVersion: v2
  apiServer:
    image: "quay.io/opendatahub/ds-pipelines-api-server:pr-50"
    argoDriverImage: "quay.io/opendatahub/ds-pipelines-driver:pr-50"
    argoLauncherImage: "quay.io/opendatahub/ds-pipelines-launcher:pr-50"
  persistenceAgent:
    image: "quay.io/opendatahub/ds-pipelines-persistenceagent:pr-50"
  scheduledWorkflow:
    image: "quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-50"
  mlmd:  
    deploy: true  # Optional component
    grpc:
      image: "quay.io/opendatahub/mlmd-grpc-server:latest"
    envoy:
      image: "registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2"
  mlpipelineUI:
    deploy: true  # Optional component 
    image: "quay.io/opendatahub/ds-pipelines-frontend:pr-50"
  objectStorage:
    minio:
      deploy: true
      image: 'quay.io/opendatahub/minio:RELEASE.2019-08-14T20-37-41Z-license-compliance'
```

Then run the following:

```bash
cd $(mktemp -d)
git clone git@github.com:opendatahub-io/data-science-pipelines.git
cd data-science-pipelines/
git fetch origin pull/50/head
git checkout -b pullrequest fbb2c121655f10f366449febcdf8f669c4204a88
oc apply -f dspa.pr-50.yaml
```

More instructions [here](https://github.com/opendatahub-io/data-science-pipelines-operator#deploy-dsp-instance) on how to deploy and test a Data Science Pipelines Application.

**Comment by dsp-developers** [comment]
### Commit Checker results:

```
**NOTE**: These are the results of the commit checker scans. 
If these are not commits from upstream kfp, then please ensure
you adhere to the commit checker formatting
```
commitchecker verson unknown
ERROR: couldn't find commits from effca01ddbe891dc2f1badfb01f30bf20283f3ef..dcbeed87bc061e1737fb14cf1ea540df5e5234f5: error executing git log: fatal: Invalid revision range effca01ddbe891dc2f1badfb01f30bf20283f3ef..dcbeed87bc061e1737fb14cf1ea540df5e5234f5
: exit status 128

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-50`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-50`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-50`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-50`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-50`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-50`

**Comment by dsp-developers** [comment]
### Commit Checker results:

```
**NOTE**: These are the results of the commit checker scans. 
If these are not commits from upstream kfp, then please ensure
you adhere to the commit checker formatting
```
```
commitchecker verson unknown
Validating 0 commits between 6623b7cc19b13b371af67b209f965afdff47dfd4...dcbeed87bc061e1737fb14cf1ea540df5e5234f5
```

**Pull request merged**